HIRS_Utils/src/main/java/hirs/data/persist/TPMBaseline.java

```diff
@@ -273,4 +274,38 @@ public final boolean removeFromBaseline(final TPMMeasurementRecord record) {
 
         return pcrRecords.remove(record);
     }
+
+    /**
+     * Checks the properties of FirmwareInfo, HardwareInfo, OSInfo, and TPMInfo and the contents of
+     * pcrRecords to determine if this instance of TPMBaseline is empty or not.
+     *
+     * @return true if baseline has no data
+     */
+    public boolean isEmpty() {
+    System.out.println("Check for empty baseline");
```

Comment:
Few thoughts for this. 
1. I'd stick with a Logger debug statement rather than println.
1. You could just return that giant boolean check directly rather than having the `if` statement.
1. A small Unit Test case added to `TPMBaseline` would be good to cover this new method.